clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2212,6 +2212,41 @@ device kernel, the attribute is ignored and it is not propagated to a kernel.
   }];
 }
 
+def SYCLIntelStallEnableAttrDocs : Documentation {
+  let Category = DocCatFunction;
+  let Heading = "intel::stall_enable";
+  let Content = [{
+Applies to a lambda function, or function definition. Requests, to the extent
+possible, that statically-scheduled clusters handle stalls using a stall-enable
+signal to freeze computation within the cluster. This attribute can be applied
+to device function/kernel and not to arbitrary device functions. This takes no
```

Comment:
Can you elaborate on this? I think this line needs to be more clear. What is device function (in device function/kernel) vs 'arbitrary device functions'

Reply:
I have updated documentation to be more clear about the attribute.